### ISSUE TITLE
zoop.com.br

### ISSUE THREAD

**Issue by brasizza** [carlosalito/flutter_zoop#2]
Esse pacote é para pinpad da zoop ?
Obrigado!

**Comment by carlosalito** [comment]
Sim, para os pin pads da zoop baseado no SDK deles, fiz esse projeto para uma maquina chinesa de vending machine (http://box247.com.br), suporte apenas para android.
